.github/workflows/e2e-test.yml

```diff
@@ -36,15 +36,15 @@ jobs:
       fail-fast:     false
       matrix:
         # Min WooCommerce | L-2 | L(atest) | beta
-        woocommerce:   [ '5.4.0', '6.3.1', '6.5.1', 'beta' ]
+        woocommerce:   [ '5.6.0', '6.3.1', '6.5.1', 'beta' ]
```

Comment:
0 is being used as the minor version here.

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/4310/commits/389c576e9c673eef7b68d6a5447dedccc013676c.